openshift/templates/secrets/README.md

```diff
@@ -50,22 +50,40 @@ oc process -f database-secrets.yaml -p ENV_NAME=prod | oc create -f -
 cd openshift/templates/app/secrets
 
 oc project $namespace_dev
-oc process -f api-secrets.yaml \
+oc process -f api-sso-secrets.yaml \
```

Comment:
Good idea to keep our secrets in separate collections.

Reply:
Agree